config.yml

```diff
@@ -4,3 +4,4 @@ with:
 metas:
   py_modules:
     - executor.py
+  workspace: workspace/
```

Comment:
i dont think this is necessary
```suggestion
```

Reply:
Hmm, If we remove the above, running `jina executor --uses config.yml` will result in:
```ValueError: Neither `metas.workspace` nor `runtime_args.workspace` is set, are you using this Executor is a Flow?```

Reply:
keep it in this way then, i think it is a bug in the core

Reply:
okie, I will merge this PR if there are no other issues